src/main/resources/kernel_cs.properties

```
@@ -40,6 +40,7 @@ NLS.description = Popis
 NLS.edit = Upravit
 NLS.errInvalidDate = Zadejte platné datum ve tvaru '${format}'. '${value}' je neplatná.
 NLS.errInvalidDecimalNumber = Zadejte platné desetinné číslo. '${value}' je neplatná.
+NLS.errInvalidIntegerNumber = Prosím, zadejte platné celé číslo. '${value}' je neplatné.
```

```suggestion
NLS.errInvalidDecimalNumber = Prosím, zadejte platné desetinné číslo. '${value}' je neplatná.
NLS.errInvalidIntegerNumber = Prosím, zadejte platné celé číslo. '${value}' je neplatné.
```
Da fehlt im oberen wohl ein "Bitte" :)